README.md

```diff
@@ -15,7 +15,34 @@ yarn add --dev storybook-addon-apollo-client
 npm install -D storybook-addon-apollo-client
 ```
 
-## As a decorator in a story
+Add the addon to your configuration in `.storybook/main.js`
+```
+module.exports = {
+  ...config,
+  addons: [
+    ...your addons
+    "storybook-addon-apollo-client",
+  ],
+};
+```
+
+
+add the following to your `.storybook/preview.js`
+
+```js
+import { MockedProvider } from '@apollo/client/testing'; // Use for Apollo Version 3+
+// import { MockedProvider } from "@apollo/react-testing"; // Use for Apollo Version < 3
+
+
+export const parameters = {
+  apolloClient: {
+    Provider: MockedProvider,
```

Comment:
It looks like this is destructured as `MockedProvider` in the decorator.
```const { MockedProvider, ...providerProps } =```
Copy+pasting this snippet from the readme will throw a warning!